Cargo.toml

```diff
@@ -1,3 +1,11 @@
 [workspace]
 members = ["crates/*"]
 resolver = "2"
+
+[workspace.package]
```

Comment:
Something about how this commit was made is causing a merge conflict

Reply:
Forgot to pull the upstream changes